src/atoms/Button/Button.stories.js

```diff
@@ -32,38 +33,46 @@ storiesOf('Button', module)
     <Flex flex={1} justifyContent="space-around" flexDirection="column">
       <View>
         <Flex flexDirection="row">
-          <View>
-            <Button
-              size={select('Sizes', sizes, 'medium')}
-              variant={select('Variants', variants, 'primary')}
-              block
-            >
-              Block Button
-            </Button>
-          </View>
+          <Space margin={[0.5, 0.5]} padding={[0.5, 0.5]}>
+            <View>
+              <Button
+                size={select('Sizes', sizes, 'medium')}
+                variant={select('Variants', variants, 'primary')}
+                block
+              >
+                Block Button
```

Comment:
Move stories with block button below the non-block ones.

Reply:
done

Reply:
done